students/mmancini/lesson03/slicing.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+
+####################################
+
+
+def slice1(in_lst):
+    print(f"slice1, original list ", in_lst)
+
+    lst_a = in_lst.copy()
+    lst_a = lst_a[-1:] + lst_a[1:-1] + lst_a[:1]
+
+    print(f"slice1, returning swap first and last, ==> ", lst_a)
+    return lst_a
+
+
+def slice2(in_lst):
+    print(f"slice2, original list ", in_lst)
+
+    lst_a = in_lst.copy()
+    lst_a = lst_a[0::2]
+
+    print(f"slice2, returning every other item removed, ==> ", lst_a)
+
+    return lst_a
+
```

Comment:
not seeing a function for "With the first and last 4 items removed, and every other item in between"